google/cloud/bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -195,35 +191,67 @@ TEST_F(InstanceAdminIntegrationTest, DeleteInstancesTest) {
 
 /// @test Verify that InstanceAdmin::ListClusters works as expected.
 TEST_F(InstanceAdminIntegrationTest, ListClustersTest) {
-  // The emulator does not support cluster operations.
-  if (UsingCloudBigtableEmulator()) {
-    return;
+  std::string id =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
+  auto config = IntegrationTestConfig(id);
+  auto instance = instance_admin_->CreateInstance(config).get();
+  auto clusters = instance_admin_->ListClusters(id);
+  for (auto const& cluster : clusters) {
+    EXPECT_NE(std::string::npos,
+              cluster.name().find(instance_admin_->project_name()));
+    EXPECT_NE(std::string::npos, cluster.name().find(id));
   }
-  std::string id = "list-clusters-test";
-  bigtable::InstanceId instance_id(id);
-  bigtable::DisplayName display_name(id);
-  std::vector<std::pair<std::string, bigtable::ClusterConfig>> clusters_config;
-  clusters_config.push_back(std::make_pair(
-      id + "-cluster1", bigtable::ClusterConfig("us-central1-f", 0,
-                                                bigtable::ClusterConfig::HDD)));
-  auto instance_config =
-      bigtable::InstanceConfig(instance_id, display_name, clusters_config)
-          .set_type(bigtable::InstanceConfig::DEVELOPMENT);
-  auto instance_details =
-      instance_admin_->CreateInstance(instance_config).get();
+  EXPECT_FALSE(clusters.empty());
+
+  instance_admin_->DeleteInstance(id);
+}
 
-  // Create clusters in an instance
-  // TODO(#422) - Implement InstanceAdmin::CreateCluster
+/// @test Verify that default InstanceAdmin::ListClusters works as expected.
+TEST_F(InstanceAdminIntegrationTest, ListAllClustersTest) {
+  std::string id1 =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
+  std::string id2 =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
 
-  // TODO(#418) - create an instance and test that its cluster is returned here.
-  auto clusters = instance_admin_->ListClusters(id);
-  for (auto const& i : clusters) {
-    auto const npos = std::string::npos;
-    EXPECT_NE(npos, i.name().find(instance_admin_->project_name()));
+  bigtable::InstanceId instance_id1(id1);
+  bigtable::InstanceId instance_id2(id2);
+
+  bigtable::ClusterId cluster_id1(id1);
+  bigtable::ClusterId cluster_id2(id2);
+  bigtable::DisplayName display_name(id1);
+
+  bigtable::ClusterConfig cluster_config =
+      bigtable::ClusterConfig("us-central1-f", 3, bigtable::ClusterConfig::HDD);
```

Comment:
This is creating two clusters in the same zone, does that configuration make sense?  Can you try to troubleshoot the integration tests against the production environment?  You should have access in the 'grass clump*" project.